traitlets/traitlets.py

```diff
@@ -864,9 +864,12 @@ def _notify_trait(self, name, old_value, new_value):
 
```

Comment:
This line seems to have wandered off, and definitely isn't where it originated. I see that the line where it was originally is removed, thanks.